Dockerfile

```diff
@@ -1,56 +1,53 @@
 ARG package=arcaflow_plugin_uperf
 
-# build poetry
-FROM quay.io/centos/centos:stream8 as poetry
+# STAGE 1 -- Build module dependencies and run tests
+# The 'poetry' and 'coverage' modules are installed and verson-controlled in the
+# quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.2.0 as build
 ARG package
 RUN dnf module -y install python39 && dnf install --setopt=tsflags=nodocs -y python39 python39-pip lksctp-tools-devel && dnf clean all \
```

Comment:
Installation of python module and packages should be removed

Reply:
Removed